.github/workflows/ci.yml

```diff
@@ -90,7 +90,7 @@ jobs:
       - uses: ./.github/actions/setup-node-env
 
       - name: Run Nuxt Playwright tests
-        run: pnpm test:playwright
+        run: pnpm test:playwright --workers=2
```

Comment:
I would if we could move this `--workers=2` argument into the `test:playwright` script itself so that we also get to benefit from this locally when we run the tests?

Also, if we're passing `cpus: 0.00` to docker-compose, do we need to pass an explicit number of workers anymore or should we just rely on the default? :thinking:

Reply:
We could move it, but it's tricky because it will depend on the limit each person has settled for docker. Here the playwright container will use half number of cores available as the number of workers. So it was 2 in my case with default settings, I increased the number of cores that Docker can use to 6 in the Docker Desktop, and now tests run with 3 workers.

<img src="https://cleanshot-cloud-fra.s3.eu-central-1.amazonaws.com/media/21958/oAOdArYUpV07EOzxapdUvagjgbjVhf4x9YLefMHm.jpeg?X-Amz-Content-Sha256=UNSIGNED-PAYLOAD&X-Amz-Security-Token=IQoJb3JpZ2luX2VjELf%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaDGV1LWNlbnRyYWwtMSJGMEQCIHCqgX%2BV5N0CmfWgHgAOIhH%2B2FJ98wfRw4OWndnTFdSVAiB%2B5jvshZZtkpmp5JcVLP9ro0Saq67avt1fQaK2jg%2F2iyqqAgjw%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F8BEAAaDDkxOTUxNDQ5MTY3NCIMs8P9h%2BOoShtJ98bpKv4BKe5SUaEz5TVIuP30J5LnzypW3tGdDt9FjeMUQOsVCDX84VFkCSAC5n1Bvb6rLQVk9Rzptszj05orHNrUvZy7u%2FVjdIbCSJmYssiovF0mmIA%2BViJFEJhkx2XzxQwrD9sOGVfIU03l6jd9c4f%2BIYBuQSS1j0zLvisOgS0Or6fJI8U%2Bf5XbpoPlAT%2FIKplJIvcZ1kJTIZ4qp9xR7k5z7imnJvSqKEr5PzAlbeeL7C7uOx7A6GVDD4ZLkWUzCvhGTrV4PxKH%2FLvY%2FzhbUDgfT5YvwjbdeH5waH6wHtSjIHjqG76UOCWjMlEhk%2BCX00NfcIcF7NcFHt8HQ6UsrQfrlqEw5OfVlgY6mwGt0vSCJYex2u3q9F82bsvT78zchJq4HS%2B7oIyAi4piglx3vTV0%2Ff9GkRzYKEG73uPOXYAmsARolZ8vayR%2BDn4ty3z4HDQ0K3aXZtTloPGOXpg%2BDh1MGm2TiGQDLQ%2BfnYQ9ybhT7DQd6PyWrUeFaURBdTswP%2Bbnrme4gmxvYsNmPM69gIc4xP%2Fct%2FpJMjYXMa1E8eNAnDA9%2BFaydg%3D%3D&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIA5MF2VVMNLHDCVNH5%2F20220718%2Feu-central-1%2Fs3%2Faws4_request&X-Amz-Date=20220718T150314Z&X-Amz-SignedHeaders=host&X-Amz-Expires=300&X-Amz-Signature=a39c170ae1912f5fab7cf97d40ce75657c67a6399151a0544ded4ff364dc49b0">

So I believe it is better to leave the command definition unchanged as everyone can pass the desired number of workers with a flag, as the CI does. Using a greater number of workers than available cores results in performance degradation.

Reply:
I might not fully understand the question, the number of workers will default to half the number of the CPU cores available, as you cited, and it's working like that within the container thanks to the `cpus: 0.00` config. Do you mean passing `--workers=2` (or more) in the command? That will not use the extra capacity in a setup of Docker with more than 4 CPUs available (like mine currently), or if we pass 3 or 4 is more than it is available in CI machines.

Reply:
Docker will use all the CPUs, but Playwrithg will use a half only. Given that CI machines only have 2 available, Playwright will use only 1, so we won't have any parallelization of tests there unless we explicitly tell it to use the 2 CPUs. It's confusing with all the environments and variables! But I hope I have explained it better here.